File: ncDados/dCaixa.vb
Namespace nsCaixa

    Public Class ColecaoCaixa
        Inherits List(Of dCaixa)
    End Class

    Public Class dCaixa

        Private _cid As Nullable(Of Integer)
        Private _nome As String
        Private _situacao As String
        Private _data As String
        Private _usuario As String

        Public Property cid() As Nullable(Of Integer)
            Get
                Return _cid
            End Get
            Set(ByVal value As Nullable(Of Integer))
                _cid = value
            End Set
        End Property

        Public Property nome() As String
            Get
                Return _nome
            End Get
            Set(ByVal value As String)
                _nome = value
            End Set
        End Property

        Public Property situacao() As String
            Get
                Return _situacao
            End Get
            Set(ByVal value As String)
                _situacao = value
            End Set
        End Property
        Public Property Data() As DateTime
            Get
                Return _data
            End Get
            Set(ByVal value As DateTime)
                _data = value
            End Set
        End Property
        Public Property usuario() As String
            Get
                Return _usuario
            End Get
            Set(ByVal value As String)
                _usuario = value
            End Set
        End Property

    End Class

End Namespace

File: ncModelos/dCaixa.cs
using System;
using System.Collections.Generic;

namespace ncModelos
{
    public class ColecaoCaixa : List<dCaixa>
    {
    }

    public class dCaixa
    {
        private int? _cid;
        private string _nome;
        private string _situacao;
        private DateTime _data;
        private string _usuario;

        public int? cid
        {
            get => _cid;
            set => _cid = value;
        }

        public string nome
        {
            get => _nome;
            set => _nome = value;
        }

        public string situacao
        {
            get => _situacao;
            set => _situacao = value;
        }

        public DateTime Data
        {
            get => _data;
            set => _data = value;
        }

        public string usuario
        {
            get => _usuario;
            set => _usuario = value;
        }
    }

}
